content/en/docs/concepts/storage/volumes.md

```diff
@@ -1160,6 +1160,33 @@ spec:
 
 More examples can be found [here](https://github.com/kubernetes/examples/tree/master/staging/volumes/vsphere).
 
+#### CSI Migration
+
+{{< feature-state for_k8s_version="v1.19" state="beta" >}}
+
+The CSI Migration feature for vsphereVolume, when enabled, shims all plugin operations
+from the existing in-tree plugin to the `csi.vsphere.vmware.com` {{< glossary_tooltip text="CSI" term_id="csi" >}} Driver. In order to use this feature, the [vSphere CSI
```

Comment:
nit: lowercase, `driver`?

Reply:
nit: The feature defaults to `enabled`? The cluster requirements are listed below.
Could you remove `when enabled`?

Reply:
This is unlike other beta features and actually defaults to off.

Reply:
Done